S/SQLite/build_tarballs.jl

```diff
@@ -7,7 +7,7 @@ version = v"3.31.1"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://sqlite.org/2020/sqlite-autoconf-3310100.tar.gz", "62284efebc05a76f909c580ffa5c008a7d22a1287285d68b7825a2b6b51949ae"),
+    ArchiveSource("https://sqlite.org/2020/sqlite-autoconf-3320300.tar.gz", "a31507123c1c2e3a210afec19525fd7b5bb1e19a6a34ae5b998fbd7302568b66"),
     ArchiveSource("https://git.archlinux.org/svntogit/packages.git/plain/trunk/license.txt?h=packages/sqlite&id=33cad63ddb1ba86b7c5a47430c98083ce2b4d86b",
```

Comment:
This should be a `FileSource`:
```suggestion
    FileSource("https://git.archlinux.org/svntogit/packages.git/plain/trunk/license.txt?h=packages/sqlite&id=33cad63ddb1ba86b7c5a47430c98083ce2b4d86b",
```
No idea how it worked before